main.php

```diff
@@ -5,16 +5,10 @@
 
 Orchestrator::AddRequirement('1.0.0', 'ldap'); // LDAP support is required to run this collector
 
-if (Utils::GetConfigurationValue('collect_person_only', 'yes') == 'yes')
-{
-	Orchestrator::AddCollector(1, 'iTopPersonLDAPCollector');
-}
-else
+$iRank = 1;
```

Comment:
Before when collect_person_only wasn't yes we were registering the collector with the 1 rank. Now we will always register it with the rank 2 ?
We might init $iRank to 0 so that iTopPersonLDAPCollector gets added at rank 1, then if needed iTopUserLDAPCollector would be at rank 2 ?

Reply:
It will always be 1 for person collector, know your basics 😜 https://3v4l.org/eCrP2

Reply:
Wooops I feel quite ashamed huhu
Thanks for the confirmation !

Reply:
And in case anyone has to relearn its lesson : https://www.php.net/manual/en/language.operators.increment.php
:)